src/service/viewer-impl.js

```diff
@@ -226,6 +226,8 @@ export class Viewer {
     dev().fine(TAG_, '- history:', this.overtakeHistory_);
 
     this.setVisibilityState_(this.params_['visibilityState']);
+    this.win.setVisibilityState = this.setVisibilityState_.bind(this);
+    this.win.getVisibilityState = this.getVisibilityState.bind(this);
```

Comment:
Curious, why do we need to add these?

Reply:
Aghhh, my git branch got messed up.  These are unrelated and should be removed.

Thanks for catching this 😃